backend/src/install/mod.rs

```diff
@@ -601,6 +601,11 @@ pub async fn uninstall_dry(
 pub async fn uninstall_impl(ctx: RpcContext, id: PackageId) -> Result<(), Error> {
     let mut handle = ctx.db.handle();
     let mut tx = handle.begin().await?;
+    crate::db::DatabaseModel::new()
```

Comment:
Makes sense that there should be a lock during the uninstall. It used to get this lock by default by getting the model on the next line. Now we have to make it more explicit.